content/docs/practices/the_zen.md

```diff
@@ -0,0 +1,118 @@
+
+## One does not simply use Histograms
+
+Histograms are powerful.
+
+Creating a correct bucket layout for your histograms is an art. To ensure usefulness of your observations and correctness of your alerts, you have come up with a meaningful bucket layout. And keep in mind that they are [cumulative](https://www.robustperception.io/why-are-prometheus-histograms-cumulative). This is also somewhat conflicting with [Instrument first, ask questions later](#instrument-first-ask-questions-later) because you need to have idea about your latencies before you even measure. To circumvent this issue you can take and iterative approach or use an event system to obtain your latency distribution.
+
+And as always, let your [SLO](https://www.youtube.com/watch?v=X99X-VDzxnw)s guide your bucket layout, create boundaries to match your SLO.
+
+The histograms underneath are just counter with labels; where bucket boundaries used as labels. Be precautious while adding additional labels to your histograms. Remember *Labels are multiplicative* and [Cardinality Matters](#cardinality-matters).
+
+---
+
+## If you can graph it, you can alert on it
+
+You can't look at dashboards 24/7. Prometheus unified metrics, dashboarding, and alerting. PromQL is the core of every Prometheus alert, and a PromQL query is the source of any graph on a dashboard. That is very powerful. Use it.
+
+## If you run it, then you should put an alert on it
+
+Always have an alert -*at least*- on presence and healthiness of your targets. You can't that rely on the things and take action that you can't observe.
+
+Avoid missing targets and unhealthy targets. All of the client libraries provides `up` metric by default. Use it.
+
+## Alerts should be urgent, important, actionable, and real
+
+[Alerts should be urgent, important, actionable, and real](https://www.robustperception.io/when-to-alert-with-prometheus). As plain as it goes.
+
+And don't over alert, alert-fatigue is real.
+
+## Symptom-based alerts for paging, caused-based for troubleshooting
+
+Similar to [Measure what users care most about](#measure-what-users-care-most-about), alert on what *really* matters. It doesn't matter if you are CPU is saturated, as long as your users don't notice. Let your [SLO](https://www.youtube.com/watch?v=X99X-VDzxnw)s guide your alerting.
+For more context [My Philosophy on Alerting](https://docs.google.com/document/d/199PqyG3UsyXlwieHaqbGiWVa8eMWi8zzAn0YfcApr8Q/edit).
+
+## Please five more minutes
+
+Prometheus alerting rules let you to specify a time duration that determines after how long an alert should start firing according to given query, which is called `FOR`. If you don't specify it a single failed scrape could cause an alert to fire. You need more tolerance. Don't make it short and always specify it. And also don't make it too long. For more information check out [Setting Thresholds on Alerts](https://www.robustperception.io/setting-thresholds-on-alerts).
```

Comment:
It's just `for` these days.

I'd recommend simplifying, and saying always at least 5 minutes

Reply:
I think "always" is too strong. I would word it more along the lines that you should only do something else if you know exactly what you are doing.